deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -503,7 +546,33 @@ <h4 class="panel-title pull-left">Capacity</h4>
                     })
                     .change(function() {
                         capacitySetting.selectedPlacements = $('.chosen-select').val()
-                    })
+                    });
+
+           if(showReplaceAlert) {
+                // include this tmpl, meaning a cluster replace is going-on, need to check its progress
+                // update every 1 min, and execute now
+                function setIntervalAndExecute(fn, t) {
+                    fn();
+                    return(setInterval(fn, t));
+                }
+                var replaceStartTime = new Date().getTime();
+                var replaceProgressInterval = setIntervalAndExecute(function() {
+                    if(new Date().getTime() - replaceStartTime > 1800000) {  // if more than 30 mins, clear it
+                        console.log('cleared time interval');
+                        clearInterval(replaceProgressInterval);
+                    }
+                    $('#activeClusterReplaceId').load(
+                            '/env/{{ env.envName }}/{{ env.stageName }}/get_replacement_progress/',
+                            function(response, status, xhr) {
+                                if(response == 'There is no on-going replacement.') {
```

Comment:
yes you are right, I need to remove this part, maybe dont need to clear the timeout event since after 30 mins, it will be cleared anyway.